src/layouts/base.njk

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>{{ meta.title }}</title>
+    <title>{%if title %}{{ title }}{% else %}{{ meta.title }}{% endif %}</title>
```

Comment:
Can we use some sort of ternary operator here? I think in Nunjucks it would be `{{ title if title else meta.title }}` or something along those lines

Reply:
👍